src/UniOneClient.php

```diff
@@ -47,6 +47,13 @@ final class UniOneClient
      */
     private string $apiKey;
 
+    /**
+     * The timeout in seconds.
+     *
+     * @var float
+     */
+    private float $timeout;
```

Comment:
Why do we add this? 🤔

Reply:
Before code refactoring, there was a method that changed the timeout. Currently, the timeout is a static 5 seconds. I delete the class field